angular-client/src/enumeration/ImportantDataType.ts

```diff
@@ -0,0 +1,5 @@
+export enum ImportantDataType {
+   DRIVER = "driver",
+   LOCATION = "location",
+   SYSTEM = "system",
+}
```

Comment:
what does this minus in a github file mean again? it's an actual no no right?

Reply:
nvm fixed

Reply:
the minus means its missing a new line at the bottom of the file